### PR TITLE
login, register and logout

### PR DESCRIPTION
**Pull request by tuanngoc1601**
## Related Tickets:

-   [Task #67882](https://edu-redmine.sun-asterisk.vn/issues/67882) : create login page
-   [Task #67883](https://edu-redmine.sun-asterisk.vn/issues/67883) : validate form login
-   [Task #67880](https://edu-redmine.sun-asterisk.vn/issues/67880) : create controller auth
-   [Task #67892](https://edu-redmine.sun-asterisk.vn/issues/67892) : create register page
-   [Task #67894](https://edu-redmine.sun-asterisk.vn/issues/67894) : validate register data
-   [Task #67889](https://edu-redmine.sun-asterisk.vn/issues/67889) : create method register controller

## WHAT this PR does?

- Tạo giao diện trang đăng nhập, đăng kí
- Validate dữ liệu từ form đăng nhập, đăng kí
- Viết controller xử lý logic đăng nhập, đăng kí
- Chức năng logout

## Notes Impacted Areas:

-   N/A.

## Attachments:

-  Login page

- Register page

### CODE REVIEW

**Comment by vutuankiet4599** [comment]
ok

**Comment by tuanngoc1601** [comment]
ready

**Comment by namnt0413** [comment]
ok

File: resources/views/auth/login.blade.php [hunk 1]
```diff
@@ -1,35 +1,27 @@
...
         <!-- Session Status -->
         <x-auth-session-status class="mb-4" :status="session('status')" />
 
-        <!-- Validation Errors -->
-        <x-auth-validation-errors class="mb-4" :errors="$errors" />
-
         <form method="POST" action="{{ route('login') }}">
             @csrf
-
+            <h2 class="text-2xl font-semibold mb-4 mt-4 text-center">Login</h2>
```
Comment (ghost) [inline resources/views/auth/login.blade.php]: localization

File: resources/views/layouts/navigation.blade.php [hunk 2]
```diff
@@ -32,14 +32,13 @@
...
                     </x-slot>
-
                     <x-slot name="content">
                         <!-- Authentication -->
                         <form method="POST" action="{{ route('logout') }}">
                             @csrf
 
                             <x-dropdown-link :href="route('logout')"
-                                    onclick="event.preventDefault();
+                                             onclick="event.preventDefault();
```
Comment (ghost) [inline resources/views/layouts/navigation.blade.php]: indent 4 spaces

File: resources/views/layouts/navigation.blade.php [hunk 3]
```diff
@@ -81,7 +80,7 @@
                     @csrf
 
                     <x-responsive-nav-link :href="route('logout')"
-                            onclick="event.preventDefault();
+                                           onclick="event.preventDefault();
```
Comment (ghost) [inline resources/views/layouts/navigation.blade.php]: indent 4 spaces

File: resources/views/auth/register.blade.php [hunk 4]
```diff
@@ -1,48 +1,55 @@
...
             @csrf
-
-            <!-- Name -->
+            <h2 class="text-2xl font-semibold mb-4 mt-4 text-center">Register</h2>
+            <!-- Username -->
             <div>
-                <x-label for="name" :value="__('Name')" />
-
-                <x-input id="name" class="block mt-1 w-full" type="text" name="name" :value="old('name')" required autofocus />
+                <x-label for="name" :value="__('Username')" />
```
Comment (ghost) [inline resources/views/auth/register.blade.php:8]: them Username vao folder lang

**Comment by tuanngoc1601** [comment]
ready

File: resources/views/auth/login.blade.php [hunk 5]
```diff
@@ -1,35 +1,27 @@
...
         <!-- Session Status -->
         <x-auth-session-status class="mb-4" :status="session('status')" />
 
-        <!-- Validation Errors -->
-        <x-auth-validation-errors class="mb-4" :errors="$errors" />
-
         <form method="POST" action="{{ route('login') }}">
             @csrf
-
+            <h2 class="text-2xl font-semibold mb-4 mt-4 text-center">Log in</h2>
```
Comment (ghost) [inline resources/views/auth/login.blade.php]: localization

**Comment by tuanngoc1601** [comment]
ready

**Pull request merged**